Sources/FluidAudio/DiarizationModels.swift

```diff
@@ -0,0 +1,299 @@
+import CoreML
+import OSLog
+
+/// Models required for diarization.
+///
+internal struct DiarizationModels {
+
+    private static var SegmentationModelFileName: String { "pyannote_segmentation.mlmodelc" }
+    private static var EmbeddingModelFileName: String { "wespeaker.mlmodelc" }
+
+    // ML models
+    internal var segmentationModel: MLModel
+    internal var embeddingModel: MLModel
+    internal var paths: ModelPaths
+
+    // Timing tracking
+    internal var downloadTime: Duration
+    internal var compilationTime: Duration
+
+    private init(
+        segmentationModel: MLModel,
+        embeddingModel: MLModel,
+        paths: ModelPaths,
+        downloadTime: Duration,
+        compilationTime: Duration
+    ) {
+        self.segmentationModel = segmentationModel
+        self.embeddingModel = embeddingModel
+        self.paths = paths
+        self.downloadTime = downloadTime
+        self.compilationTime = compilationTime
+    }
+}
+
+// ---------------------------
+// MARK: - Downloading models.
+// ---------------------------
+
+extension DiarizationModels {
+
+    private static var HuggingFaceRepoPath: String { "bweng/speaker-diarization-coreml" }
```

Comment:
```suggestion
    private static var HuggingFaceRepoPath: String { "FluidInference/speaker-diarization-coreml" }
```

https://huggingface.co/FluidInference/speaker-diarization-coreml

Not your change but could just update here

Reply:
Done